arch/arm/src/rp2040/rp2040_serial.c

```diff
@@ -884,8 +884,7 @@ static bool up_rxavailable(struct uart_dev_s *dev)
 
 static void up_send(struct uart_dev_s *dev, int ch)
 {
-  struct up_dev_s *priv = (struct up_dev_s *)dev->priv;
-  up_serialout(priv, RP2040_UART_UARTDR_OFFSET, (uint32_t)ch);
+  up_putc(ch);
```

Comment:
how to handle the non default device?

Reply:
Done, use arm_lowputc() instead of up_putc()